clean_project/construct_dict.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""
+Construct a JSON file used to indexing all words in a config_file.yaml
+"""
+import argparse
+import codecs
+import json
+import os
+import yaml
+
+from collections import defaultdict
+from clean_file import sanitize
+
+
+def construct_dictionary(yaml_path, path_json_file):
+    """
+    Construct the dictionary, given a configuration files
```

Comment:
also, not a `dictionary`, but an `index`: if you say dictionary, you're exposing the implementation, which you don't want. If you say that you're building a dictionary, implicitly people reading the documentation will rely on the fact that it's a dictionary. 

Example:
I read the docs and read that it's a dictionary. Knowing that, I do all sorts of direct accesses to the index because I know that it's a dictionary.
One day you change the implementation for whatever reason and make it... don't know, a list (it's dumb, but let's assume so for the sake of the example).
My code will now run MUUUUCH more slowly because instead doing O(1) access to the entry, you maybe scan the whole list until you find the element I requested.

It's a very dumb case limit, but that's the idea of why you shouldn't expose the implementation details in docs or **public** class/methods